website/docs/outposts/index.mdx

```diff
@@ -2,39 +2,67 @@
 title: Outposts
 ---
 
-An outpost is a single deployment of an authentik component, which can be deployed in a completely separate environment:
+An outpost is a single deployment of an authentik component, essentially a service, that can be deployed anywhere that allows for a connection to the authentik API.
+
+An outpost is required if you use any of the following types of providers with your application:
 
 -   [LDAP Provider](../providers/ldap/index.md)
 -   [Proxy Provider](../providers/proxy/index.md)
 -   [RADIUS Provider](../providers/radius/index.md)
 -   [RAC Provider](../providers/rac/index.md)
 
-![](outposts.png)
+These types of providers use an outpost for increased flexibility and speed. Instead of the provider interacting directly with the business logic in the authentik Core to process all transactions, these providers use an outpost to handle the logic, which provides better isolation and performance improvements (because the outpost is written in Go while the provider is written in Python).
+
+An additional advantage of using an outpost is that outposts, like authentik itself, do not require access to the wider internet. Transactions between the application, the provider, and the outpost occur via the authentik API, and support single sign-on operations in firewalled or airgapped deployments and offline connections to remote machines that are not on the internet.
+
+An outpost is given permissions to access the authentik API using a service account and token, both of which are auto-generated when you create a new outpost. The outpost is granted rights to only the one app/provider pair.
+
+Any change made to the outpost's associated app or provider immediately triggers an event to update the configuration data stored on the outpost, via websockets. Websockets are also used to send authentik auto-scheduled health-checks on the outpost.
+
+## Create and configure an outpost
+
+    1. To create a new outpost, log in to authentik as an administrator, and open to the Admin interface.
+
+    2. Navigate to **Applications --> Outposts** and then click **Create**.
+
+![](outpost-create.png)
+
+    3. Define the following values:
+
+    - **Name**: a name for the new outpost
+    - **Type**: select the provider type (Proxy, LDAP, Radius, RAC)
+    - **Integration** (_optional_): select either your [Docker or Kubernetes connection](#more-about-outpost-integrations)
+    - **Applications**: select the applications that you want the outpost to communicate with
```

Comment:
```suggestion
    - **Applications**: select the applications that you want the outpost to serve
```

(could also use `provide` instead of `serve`?)